doc/source/user/feature-classification/feature-matrix-member.ini

```diff
@@ -0,0 +1,112 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+[driver.elb]
+title=Enhanced LB Provider
+link=https://docs.otc.t-systems.com/api/elb/elb_zq_0000.html
+
+[driver.vlb]
+title=Dedicated LB Provider
+link=https://docs.otc.t-systems.com/api/elb/elb_zq_0000.html
+
+# Note: These should be in the order they appear in a create call.
+
+[operation.admin_state_up]
+title=admin_state_up
+status=mandatory
+notes=Enables and disables the member.
+cli=openstack loadbalancer member create [--enable | --disable] <pool>
+driver.elb=partial
+driver.vlb=partial
+driver-notes.elb=Only active
+driver-notes.vlb=Only active
+
+[operation.address]
+title=address
+status=mandatory
+notes=The IP address for the member.
+cli=openstack loadbalancer member create --address <ip_address> <pool>
+driver.elb=complete
+driver.vlb=complete
+
+[operation.backup]
+title=backup
+status=optional
+notes=True if the member is a backup member server.
+cli=openstack loadbalancer member create [--enable-backup] <pool>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.batch]
+title=Batch update members
+status=mandatory
+notes=Ability to update the members of a pool in one API call.
+driver.elb=missing
+driver.vlb=missing
+
+[operation.monitor_address]
+title=monitor_address
+status=optional
+notes=An alternate IP address used for health monitoring a backend member.
+cli=openstack loadbalancer member create [--monitor-address <monitor_address>] <pool>
+driver.elb=missing
+driver.vlb=missing
+
+[operation.monitor_port]
+title=monitor_port
+status=optional
+notes=An alternate protocol port used for health monitoring a backend member.
+cli=openstack loadbalancer member create [--monitor-port <monitor_port>] <pool>
+driver.elb=missing
+driver.vlb=missing
+driver-notes.vlb=Technically can be implemented through monitor_port of the HealthMonitor
```

Comment:
monitor_port is missed in Octavia Health Monitor API. It present in elb v2/ elb v3 API.